runtime/test/planner-tests.js

```diff
@@ -23,7 +23,9 @@ let SearchTokensToParticles = require('../strategies/search-tokens-to-particles.
 let GroupViewConnections = require('../strategies/group-view-connections.js');
 let CombinedStrategy = require('../strategies/combined-strategy.js');
 let FallbackFate = require('../strategies/fallback-fate.js');
-
+let MessageChannel = require('../message-channel.js');
+let InnerPec = require('../inner-pec.js');
```

Comment:
ahh, I think this needs to be `inner-PEC.js`

also filed https://github.com/PolymerLabs/arcs/issues/488 about renaming these.

Reply:
right, thank you! i don't understand how this could happen, i'm pretty much sure i copied it from another file... nm, all seem good now!